botfront/private/default-settings.docker-compose.ci.json

```diff
@@ -12,7 +12,7 @@
             "bfApiHost": "http://botfront-api:8080",
             "rootUrl": "http://botfront:3000",
             "actionsServerUrl": "http://actions:5055/webhook",
-            "defaultEndpoints": "nlg:\n  type: 'rasa_addons.core.nlg.GraphQLNaturalLanguageGenerator'\n  url: '{ROOT_URL}/graphql'\naction_endpoint:\n  url: '{ACTIONS_URL}'\ntracker_store:\n  store_type: rasa_addons.core.tracker_stores.AnalyticsTrackerStore\n  url: '{BF_API_HOST}'\n  project_id: '{BF_PROJECT_ID}'",
+            "defaultEndpoints": "nlg:\n  type: 'rasa_addons.core.nlg.GraphQLNaturalLanguageGenerator'\n  url: '{ROOT_URL}/graphql'\naction_endpoint:\n  url: '{ACTIONS_URL}'\ntracker_store:\n  store_type:  'rasa_addons.core.tracker_stores.botfront.BotfrontTrackerStore'\n url: '{ROOT_URL}/graphql'\n  project_id: '{BF_PROJECT_ID}'",
```

Comment:
Don't forget to do this on the other default-settings.**.json files on EE.

Reply:
done in : https://github.com/botfront/botfront-ee/pull/98